sentry/src/main/java/io/sentry/instrumentation/file/FileInputStreamInitData.java

```diff
@@ -0,0 +1,27 @@
+package io.sentry.instrumentation.file;
+
+import io.sentry.IHub;
+import io.sentry.ISpan;
+import java.io.File;
+import java.io.FileInputStream;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+final class FileInputStreamInitData {
+
+  final @Nullable File file;
+  final @Nullable ISpan span;
+  final @NotNull FileInputStream delegate;
+  final @NotNull IHub hub;
+
+  public FileInputStreamInitData(
```

Comment:
```suggestion
  FileInputStreamInitData(
```

if the class is package private, I guess we don't need to make it public here